suites/reef/rgw/tier-2_rgw_multisite_regression_extended.yaml

```diff
@@ -268,6 +268,23 @@ tests:
       polarion-id: CEPH-83575227
       module: exec.py
       name: get shared realm info on secondary
+
+  - test:
+      name: test radosgw-admin negative commands
+      desc: Test negative scenarios for radosgw-admin commands in RGW multisite setup.
+      polarion-id: CEPH-83575397
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-pri:
+          config:
+            set-env: true
```

Comment:
Can you please confirm, since we are setting set-env: true here as well as in next testcsae.
it will not cause any failure.